baggysoapshop/settings.py

```diff
@@ -367,13 +370,24 @@ def location(filename):
 OSCAR_DASHBOARD_NAVIGATION += [
     {
         'label': 'Shipping',
+        'icon': 'fas fa-shipping-fast',
         'children': [
             {
                 'label': 'Shipping',
                 'url_name': 'dashboard:shipping-method-list',
             },
          ]
     },
+    {
+        'label': _('PayPal'),
```

Comment:
:+1: Nice! I should have done this with my labels in the Shipping section above :see_no_evil: